adrs/001-node-rust-foreign-function-interface/napi-prototype.md

```diff
@@ -0,0 +1,142 @@
+## [Back](001-node-to-rust-foreign-function-interface.md)
+
+## Napi-rs prototype
+___
+
+1. Install the napi CLI
+
+```bash
+npm install -g @napi-rs/cli
+```
+
+2. Create a new napi project
+
+```bash
+napi new napi-prototype
```

Comment:
```suggestion
-> napi-prototype: napi new napi-prototype
```

Reply:
We are not inside the directory, we are creating it with this command